### DISCUSSION TITLE
How do we load emmet or other plugins?

### DISCUSSION THREAD

**Discussion by ItsMeStevieG** [suren-atoyan/monaco-react#202] [Q&A] [answered]
Any help would be appreciated.

**Comment by suren-atoyan** [comment, marked as answer]
`emmet` is not part of `monaco`, and there are no extensions for `monaco`, but there is always a way. check [this](https://codesandbox.io/s/simple-usage-forked-34ftw?file=/src/App.js) out.

**Reply by ItsMeStevieG** [reply]
Thanks for the the sandbox code Example! Love what you are doing here :)

**Reply by ItsMeStevieG** [reply]
My first all-in one attempt: https://codesandbox.io/s/vscode-monaco-editor-with-theme-and-emmet-jlmo8?file=/src/Components/App/App.js

**Reply by suren-atoyan** [reply]
[multi-model editor](https://github.com/suren-atoyan/monaco-react#multi-model-editor) can be useful here

**Reply by ItsMeStevieG** [reply]
Would love to see you take what I've written and make a self hosted version with file manager and imports similar to codesandbox (only self hosted)
